src/constants/tokens.ts

```diff
@@ -515,6 +515,16 @@ export function nativeOnChain(chainId: number): NativeCurrency | Token {
   return (cachedNativeCurrency[chainId] = nativeCurrency)
 }
 
+export function getSwapCurrencyId(currency: Currency): string {
+  if (currency.isToken) {
+    return currency.address
+  }
+  if (currency.isNative) {
```

Comment:
a currency can only be `isToken` or `isNative` 
```suggestion
return `NATIVE_CHAIN_ID`
```